deepchem/splits/test_scaffold_splitter.py

```diff
@@ -0,0 +1,134 @@
+import unittest
+from unittest import TestCase
+
+import numpy as np
+import deepchem as dc
+from deepchem.splits import Splitter
+from deepchem.utils import ScaffoldGenerator
+from deepchem.utils.save import log
+from rdkit import Chem
+
+
+def generate_scaffold(smiles, include_chirality=False):
+  """Compute the Bemis-Murcko scaffold for a SMILES string."""
+  mol = Chem.MolFromSmiles(smiles)
+  engine = ScaffoldGenerator(include_chirality=include_chirality)
+  scaffold = engine.get_scaffold(mol)
+  return scaffold
+
+
+class ScaffoldSplitter(Splitter):
```

Comment:
We should import `ScaffoldSplitter` from `deepchem.splits.splitters` and remove the class from this testing file.

Reply:
I am getting `ModuleNotFoundError` when I am trying to import the Splitter. Should I build this package somehow before doing the import?

Reply:
No build should be required here. How are you trying to run the tests? I think you might be trying to run the file directly which would cause this error. Instead, run the test through `nose` and you'll see these imports resolve correctly with

`nosetests -v deepchem/splits/test_scaffold_splitter.py`

Reply:
Okay great, that's how you run it. Removed